backend/notes/views.py

```diff
@@ -35,4 +35,18 @@ def post(self, request):
 
 class CreateUserView(generics.CreateAPIView):
     queryset = User.objects.all()
-    serializer_class = UserSerializer
\ No newline at end of file
+    serializer_class = UserSerializer
+
+class LogInView(APIView):
+    def post(self, request):
+        username = request.data.get('login')
+        password = request.data.get('password')
+        userObjects = User.objects.all().values()
+        
+        correctCredentials = False
+        for user in userObjects:
```

Comment:
This solution is very uneffective, imagine what will happened if we'll have 1.000.000 users in the database and every time we'll need to search through everyone to find the right one or respond that it doesn't exist.

Instead of using `User.objects.all().values()` please use User.objects.get() and put in this method the primary key of the user more info here https://docs.djangoproject.com/en/4.2/topics/db/models/